runtime-light/component/image.h

```diff
@@ -4,7 +4,6 @@
 
 #pragma once
 
-#include "runtime-light/header.h"
 #include "runtime-light/stdlib/rpc/rpc-context.h"
 
 struct ImageState {
```

Comment:
Let's also delete `c_linear_mem` since it isn't used

Reply:
It's actually used in the compiler. If we want to remove this, let's do it in a separate PR.